### PR TITLE
New setup wizard

### PR DESCRIPTION
**Pull request by aenario**
Styles isnt over & some trouble at the end of wizard.

THis PR is mostly for code review.

### CODE REVIEW

File: www-src/app/lib/base_view.coffee [hunk 1]
```diff
@@ -2,7 +2,8 @@ module.exports = class BaseView extends Backbone.View
 
     template: ->
 
-    initialize: ->
+    initialize: (options) ->
```
Comment (kosssi) [inline www-src/app/lib/base_view.coffee]: You can juste write:

    initialize: (@options) ->

File: www-src/app/assets/fonts/fonts.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ * - font-style: italic / font-weight: bold
+ */
+@font-face {
+    font-family: 'Source Sans Pro';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Source Sans Pro'),
+         local('SourceSansPro-Regular'),
+         url('./sourcesanspro-regular.woff2') format('woff2'),
+         url('./sourcesanspro-regular.woff') format('woff');
```
Comment (nono) [inline www-src/app/assets/fonts/fonts.css]: @aenario I think we can remove the woff2 support. It's interesting for a webapp as it saves some bandwidth. Bit for a cordova app, the font is already in the package and is better to not include it twice.

File: www-src/app/replicator/filesystem.coffee [hunk 3]
```diff
@@ -193,3 +193,15 @@ __chromeSafe = ->
...
+        model: "Debug Browser"
+        platform: "Browser"
+        uuid: '2323-5343-2323-3232'
+        serial: '2323-5343-2323-3232'
+        version: '1.2'
+        manufacturer: 'Debug inc.'
+        isVirtual: false
+        cordova: '5.3'
+
+    $(window).bind 'click', (e) -> $(e.target).trigger 'tap'
```
Comment (nono) [inline www-src/app/replicator/filesystem.coffee]: s/bind/on/ ?

File: www-src/app/views/login.coffee [hunk 4]
```diff
@@ -1,71 +1,62 @@
...
+cleanUrl = require '../lib/cleanurl'
 
 module.exports = class LoginView extends BaseView
 
-    className: 'list'
-    template: require '../templates/login'
+    menuEnabled: false
+    className: ->
+        classes = ['wizard-step']
+        classes.push @options?.step if @options?.step
```
Comment (nono) [inline www-src/app/views/login.coffee]:
```coffee
classes.push @options.step if @options?.step
```

File: www-src/app/lib/base_view.coffee [hunk 5]
```diff
@@ -10,9 +10,20 @@ module.exports = class BaseView extends Backbone.View
...
         @
 
+    bindRefs: ->
+        for ref, selector of @refs
+            @[ref] = @$ selector
+
+    setState: (key, value) ->
+        clearTimeout @dirtyTimeout
+        @[key] = value
+        @dirtyTimeout = setTimeout @render.bind(@), 1
```
Comment (nono) [inline www-src/app/lib/base_view.coffee]: @aenario maybe you can use a fat arrow when declaring `render` to avoid the `.bind(@)` here.

**Comment by frankrousseau** [comment]
ping @aenario, @kosssi  It's the end of the sprint. Can we merge this PR?

**Comment by kosssi** [comment]
@aenario you should merge in development not in master branch and can you rebase it ?

**Pull request closed without merging**